src/components/TabPanel/TabPanel.tsx

```diff
@@ -1,24 +1,53 @@
 import React from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
 
+import { Button } from '@material-ui/core';
```

Comment:
Fant vi ut noe ang `import { Button } from '@material-ui/core';` vs `import Box from '@material-ui/core/Box';`? Her bytter vi notasjon på neste linje igjen

Reply:
If you're using ES6 modules and a bundler that supports tree-shaking (webpack >= 2.x, parcel with a flag) you can safely use named imports and expect only a minimal set of Material-UI components in your bundle:
https://material-ui.com/guides/minimizing-bundle-size/

Reply:
Da kan vi prøve å holde oss til named imports fremover da